### PR TITLE
lodash: signatures of _.isEmpty have been changed

### PR DESCRIPTION
**Pull request by chrootsu**
**Please merge https://github.com/DefinitelyTyped/DefinitelyTyped/pull/7607 before**

https://lodash.com/docs#isEmpty
https://lodash.com/docs#_ (description of the chainable wrapper)

### CODE REVIEW

File: lodash/lodash.d.ts [hunk 1]
```diff
@@ -10445,6 +10446,13 @@ declare module _ {
         isEmpty(): boolean;
     }
 
+    interface LoDashExplicitWrapperBase<T,TWrapper> {
+        /**
+         * @see _.isEmpty
+         */
+        isEmpty(): LoDashExplicitWrapper<boolean>;
```
Comment (DomiR) [inline lodash/lodash.d.ts]: This should probably not return Wrapper as it is not chainable:

From https://github.com/lodash/lodash/blob/3.10.1/doc/README.md#_isemptyvalue
**The wrapper methods that are not chainable by default are:**
add, attempt, camelCase, capitalize, ceil, clone, cloneDeep, deburr, endsWith, escape, escapeRegExp, every, find, findIndex, findKey, findLast, findLastIndex, findLastKey, findWhere, first, floor, get, gt, gte, has, identity, includes, indexOf, inRange, isArguments, isArray, isBoolean, isDate, isElement, **isEmpty**, isEqual, isError, isFinite isFunction, isMatch, isNative, isNaN, isNull, isNumber, isObject, isPlainObject, isRegExp, isString, isUndefined, isTypedArray, join, kebabCase, last, lastIndexOf, lt, lte, max, min, noConflict, noop, now, pad, padLeft, padRight, parseInt, pop, random, reduce, reduceRight, repeat, result, round, runInContext, shift, size, snakeCase, some, sortedIndex, sortedLastIndex, startCase, startsWith, sum, template, trim, trimLeft, trimRight, trunc, unescape, uniqueId, value, and words

Comment (chrootsu) [inline lodash/lodash.d.ts]: @DomiR thanks for your review. It is a case when the chaining used explicitly, e.g. `_('').chain().isEmpty()`. So the returning of wrapper is correct here.

Comment (DomiR) [inline lodash/lodash.d.ts]: You're absolutely right, my bad. Doesn't make much sense to me, but the return type is a wrapper indeed :+1:

**Comment by vvakame** [comment]
@chrootsu please resolve conflict.

**Comment by chrootsu** [comment]
@vvakame done.

**Pull request merged**